releasenotes/notes/migrate-to-qiskit-nature-0.5.0-96e90cd48f36d731.yaml

```diff
@@ -0,0 +1,23 @@
+---
+features:
+  - |
+    Full support for Qiskit Nature 0.5.0.
```

Comment:
At this point, we actually depend on Qiskit Nature 0.5.2 (we had to bump the version because quantum-serverless depends on it, and the minimum version tests [aren't smart enough](https://github.com/Qiskit-Extensions/circuit-knitting-toolbox/pull/122#issuecomment-1530600696) to disambiguate that they are only needed for the notebooks).  I think it makes sense just to replace 0.5.0 with 0.5 everywhere in this release note.